src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -1706,10 +1707,12 @@ static void FillScanInfo(Eventinfo *lf,cJSON *scan_id,cJSON *name,cJSON *descrip
     if(scan_id) {
         char value[OS_SIZE_128];
 
-        if(scan_id->valueint >= 0){
-            sprintf(value, "%d", scan_id->valueint);
-        } else if (scan_id->valuedouble) {
-            sprintf(value, "%lf", scan_id->valuedouble);
+        if(cJSON_IsNumber(scan_id)){
```

Comment:
Could you check the styling of all conditional statements?  We use the following [guide](https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C%29)